src/App.tsx

```diff
@@ -15,37 +15,126 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+const showList = (arr: string[]) =>
+  arr.map(good => (
+    <li key={good} data-cy="Good">
+      {good}
+    </li>
+  ));
```

Comment:
This function name 'showList' does not start with a verb, which is not in line with the [CODE STYLE] checklist. Consider renaming it to something like 'renderList' to clearly indicate that it is responsible for rendering a list.